CODE_OF_CONDUCT.md

```diff
@@ -27,7 +27,7 @@ Examples of unacceptable behavior by participants include:
 * Trolling, insulting/derogatory comments, and personal or political attacks
 * Public or private harassment
 * Publishing others' private information, such as a physical or electronic
-  address, without explicit permission
+  addressess, without explicit permission
```

Comment:
The plural of `address` is `addresses`. But also singular seems just fine to me here.

Reply:
Note that the `chk_spelling` CI job is even failing due to this.

Reply:
Can you just remove the change?